library/agent/hooks/instrumentation/loadHook.packages.test.ts

```diff
@@ -0,0 +1,57 @@
+import * as mod from "node:module";
+import * as t from "tap";
+import { createTestAgent } from "../../../helpers/createTestAgent";
+import { ReportingAPIForTesting } from "../../api/ReportingAPIForTesting";
+import { Token } from "../../api/Token";
+import { applyHooks } from "../../applyHooks";
+import { Hooks } from "../Hooks";
+import { registerNodeHooks } from "./index";
+import type { Event } from "../../api/Event";
+
+// Using a separate file because we can only call registerNodeHooks once
```

Comment:
Only for future reference: You can still update the instrumentation instructions (which packages to patch) using applyHooks() or directly calling the setter functions from the instructions module